static/app/views/settings/projectSecurityAndPrivacy/index.tsx

```diff
@@ -32,10 +34,13 @@ export default function ProjectSecurityAndPrivacy({organization, project}: Props
   const apiMethod = 'PUT';
   const title = t('Security & Privacy');
 
+  const hasAccess = hasEveryAccess(['project:write'], {organization, project});
+
   return (
     <Fragment>
       <SentryDocumentTitle title={title} projectSlug={projectSlug} />
       <SettingsPageHeader title={title} />
+      <PermissionAlert access={['project:write']} project={project} />
```

Comment:
```suggestion
      <PermissionAlert project={project} />
```
the default `access` in PermissionAlert is `['project:write']`